views/partials/dashboards/breaches-dash.hbs

```diff
@@ -36,9 +36,10 @@
       <h2 class="pref-headline">{{ getString "email-addresses-title" }}</h2>
       {{> dashboards/manage-email-link variableClass="hide-mobile"}}
     </div>
-    <div class="email-cards flx flx-col jst-cntr">
+    <div class="email-cards flx flx-col jst-cntr {{#if experimentFlags.treatmentBranch}}experiment{{/if}}">
```

Comment:
Note - I wasn't able / didn't see the value in persisting the `experimentFlags` data all the way down to each `verifiedEmail` object. This solution still prints it out, but it is hidden (via CSS) by default.